packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -160,6 +224,8 @@ export class ApolloServer extends ApolloServerBase {
             endpoint: path,
```

Comment:
Is it possible to make this `endpoint` configurable? or should we be using `tabs` like:
```
{
  tabs: [{ endpoint: '/graphql' }]
}
```
Though it doesn't you can set a subscription endpoint for a tab.